app/controllers/upvotes_controller.rb

```diff
@@ -26,6 +26,7 @@ def destroy
     upvote = answer.upvotes.find(params[:id])
     if current_user.has_upvoted? answer
       upvote.destroy
+      redirect_to "/terms/#{params[:term_id]}"
     end
```

Comment:
actually we might as well just delet ethe whole destroy method we're not actually using it anywhere (had a weird routing problem so I put all the destroy logic in the show method for upvotes/downvotes) (we can try refactor this with rails path links instead as well? :p)